ui/src/components/Tasks/validate.js

```diff
@@ -24,15 +24,15 @@ const validate = (values, props) => {
   // here we update the resolution limits based on the energy the typed in the form,
   // the limits come from a table sent by the client
 
-  const validFname = /^[\w#[\]{}-\-]+$/u.test(props.filename);
+  const validFname = /^[-_#{}[\]\w]+$/u.test(props.filename);
```

Comment:
Initial regex was `/^[-\w\-\#\_\{\}\[\]]+$/`.

- None of the characters need to be escaped except `]`.
- The dash was provided unescaped and escaped. I've kept the unescaped one and kept it at the start of the character set (i.e. `[...]`) so it is never considered as a range (e.g. `a-z`).